C/CLFFT/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "CLFFT"
+version = v"2.12.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/clMathLibraries/clFFT.git", "1e4833f060976971c4df4b54b1b9ad1620aaf1fb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+install_license ./clFFT/LICENSE
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -S ./clFFT/src -B ./clFFT/build
+cmake --build ./clFFT/build --target install -j${nproc}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
```

Comment:
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=22752&view=logs&j=fe700123-3cf3-52ba-7dd2-386454acedcf&t=be72c861-d521-5b5e-9928-86c0a097112b&l=514
```
┌ Warning: /agent/_work/1/s/C/CLFFT/build/x86_64-linux-gnu/13HbzKwv/x86_64-linux-gnu-libgfortran3-cxx11/destdir/lib64/libStatTimer.so.2.12.2 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/dyIhw/src/auditor/compiler_abi.jl:247
```
```suggestion
platforms = expand_cxxstring_abis(platforms)
```